tests/Event/Events/AudienceUpdatedTest.php

```diff
@@ -5,36 +5,24 @@
 namespace CultuurNet\UDB3\Event\Events;
 
 use CultuurNet\UDB3\Event\ValueObjects\Audience;
-use CultuurNet\UDB3\Event\ValueObjects\AudienceType;
+use CultuurNet\UDB3\Model\ValueObject\Audience\AudienceType;
 use PHPUnit\Framework\TestCase;
 
 class AudienceUpdatedTest extends TestCase
 {
-    /**
-     * @var string
-     */
-    private $itemId;
+    private string $itemId;
 
-    /**
-     * @var AudienceType
-     */
-    private $audienceType;
+    private AudienceType $audienceType;
```

Comment:
This can become a local variable instead of a property. Or even directly passed into `new Audience` on line 27.